### ISSUE TITLE
Death feels too punishing

### ISSUE THREAD

**Issue by wonderstarcomet** [BlueStaggo/MCDiverge#28]
Not sure if this issues page is for high-level design decisions but it seems like there's no better place to put this.

I know dropping your inventory on death is a base MC mechanic but it's aggravating in this mod nonetheless. If you goof up and make an error die in combat (relatively common because you can't easily run away from monsters anymore and regeneration has been significantly nerfed) far from your spawn point, there is essentially nothing you can do. Even very simple lapses in judgement can cost you literal hours of progress. It makes any kind of exploration terrifying unless you play like a paranoid schizophrenic with the debug menu open 24/7.

Some suggestions for making death not rage-inducing:
* Some equivalent of keepInventory
* Some kind of item that lets you see your death location
* Just straight up printing your death location to the console
* Some kind of corpse/gravestone mechanic on top of everything else

**Comment by BlueStaggo** [comment]
Definitely will do something about this. The game will get more difficult and get more powerful content so it would be devastating to die like that. I'll definitely fix it up (it has already been in consideration).

**Comment by Sunflames0** [comment]
In the discord staggo said he would push a solution, something like not dropping your tools and armor but dropping the rest

**Comment by BlueStaggo** [comment]
Yes, in the beta rework, your equipment is now kept after death and other items like blocks and random junk items still drop upon death. Hopefully this serves as a good compromise.
